apps/molgenis-components/src/queryEmx2/queryEmx2.ts

```diff
@@ -474,33 +474,28 @@ class QueryEMX2 {
   _createFilterFromPath(path: string, operator: string, value: any) {
     const valueArray = Array.isArray(value) ? value : [value];
 
-    for (const value of valueArray) {
-      /** reverse the path, so we can build it from the inside out */
-      const reversedPathParts = path.split(".").reverse();
-      let graphqlValue = typeof value === "boolean" ? `${value}` : `"${value}"`;
+    /** set the correct operators for graphQl */
+    if (operator === "in") {
+      operator = "equals";
```

Comment:
Don't do input modification.
I suggest making a getGqlOperator function for this